pom.xml

```diff
@@ -360,13 +365,46 @@
                 <artifactId>api</artifactId>
                 <version>${strimzi.version}</version>
             </dependency>
+            <dependency>
```

Comment:
We want to avoid these libraries becoming prod dependencies by mistake.  I think it this case, I'd manage these at the child pom level, or make their scope test here, to make the intent clear.